tools/static-assets/server/boot.js

```
@@ -411,6 +407,12 @@ const loadServerBundles = Profile("Load server bundles", async function () {
     const scriptPath =
         parsedSourceMaps[absoluteFilePath] ? absoluteFilePath : fileInfoOSPath;
 
+    // \n is necessary in case final line is a //-comment
+    // The sourceURL pragma improves source attribution in V8 profilers
+    // and debugging contexts for dynamically evaluated code.
+    wrapParts.push("){", code, "\n//# sourceURL=" + scriptPath + "\n})");
```

ESLint

[error] Replace `"){",·code,·"\n//#·sourceURL="·+·scriptPath·+·"\n})"` with `'){',·code,·'\n//#·sourceURL='·+·scriptPath·+·'\n})'`

(prettier/prettier)